website/cue/reference/components.cue

```diff
@@ -139,7 +139,7 @@ components: {
 	}
 
 	// `#ComponentKind` represent the kind of component.
-	#ComponentKind: "sink" | "source" | "transform"
+	#ComponentKind: "sink" | "source" | "transform" | "enrichment_table"
```

Comment:
When discussing "components" in Vector, I always considered them to be sinks/sources or transforms, and all of them are part of a "pipeline".

Are we sure we want to muddy the waters here by also chucking enrichment tables within that category? I'd personally vote against that decision.

It seems to me the closer analogy to enrichment tables are environment variables: we provide a way to access the data they store, but it's not something you directly hook into a pipeline. Should they be in a new category of "data providers" or something similar?

I realize this is our Cue docs, and not exposed publicly, but from experience, once terminology gets muddied, it seeps through in other places, and before you know it, when discussing "components", half of the people in a meeting think you're talking about enrichment tables, and the other half consider those to be of a different kind, making it more difficult to have precise discussions on the topic.

Reply:
I just noticed @binarylogic made a similar comment:

> Thanks for taking a stab at this! I'd like to see us properly document the root level enrichment_tables concept instead of trying to wedge it in as a new component type.

Maybe we forgot to remove it from this list?

Reply:
@JeanMertz Yep, just forgot to remove it from this list. Enrichment tables are no longer listed as a component type.